conf/base.config

```diff
@@ -11,28 +11,38 @@
 
 process {
 
-  cpus = { check_max( 8 * task.attempt, 'cpus' ) }
+  cpus = { check_max( 1 * task.attempt, 'cpus' ) }
   memory = { check_max( 7.GB * task.attempt, 'memory' ) }
   time = { check_max( 4.h * task.attempt, 'time' ) }
 
   errorStrategy = { task.exitStatus in [143,137,104,134,139] ? 'retry' : 'finish' }
   maxRetries = 1
```

Comment:
It might help to raise `maxRetries` if there are connection timeouts to the NCBI server